docs/Scripting.md

```diff
@@ -35,10 +35,10 @@ Any object that exposes field or property members with the same name as @-prefix
  - RedisKey
  - RedisValue
 
-StackExchange.Redis handles Lua script caching internally. It automatically transmits the Lua script to redis on the first call to 'ScriptEvaluate'. For further calls of the same script only the hash with [`EVALSHA`](https://redis.io/commands/evalsha) is used.
+StackExchange.Redis handles Lua script caching internally. It automatically transmits the Lua script to redis on the first call to 'ScriptEvaluate'. For further calls of the same script [`EVAL`](https://redis.io/commands/eval) is used instead of [`EVALSHA`](https://redis.io/commands/evalsha), due to resiliency concerns.
 
 For more control of the Lua script transmission to redis, `LuaScript` objects can be converted into `LoadedLuaScript`s via `LuaScript.Load(IServer)`.
```

Comment:
I was wondering if we should get rid of this paragraph at all, since the loaded script abstraction doesn't evaluate the script by its hash anymore. Any thoughts on this ?

Reply:
This comment is irrelevant.